### ISSUE TITLE
Conflict with add-ons probably relying on message ID

### ISSUE THREAD

**Issue by bege10** [cleidigh/EditEmailSubject-MX#88]
I use the add-on QNote that adds notes to messages like XNote does. I guess they rely on the message IDs.
Changing the subject with EES changes the message ID, thus the notes get lost.
I am afraid there is no solution for this conflict, nevertheless I want to mention it here.

**Comment by jobisoft** [comment]
Oh, I think I change the header message id to get around sync issues. I have worked on improved IMAP code and will release a beta soon. Will try to not change the header id anymore. Would you be avail for tests?

**Comment by bege10** [comment]
> Would you be avail for tests?

Yes, sure. It may take a few days because of work and health issues.
